canonical-web-and-design/jaas-dashboard
# Remove units view and panel from model details

## Done

Remove units view and panel from model details

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Go to model details and verify that the units view has been removed.

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1594
